Makefile

```diff
@@ -2,37 +2,40 @@
 CHART := java
 RELEASE := chart-${CHART}-release
 NAMESPACE := chart-tests
-TEST := ${RELEASE}-test-service
+TEST := ${RELEASE}-${CHART}-test
 ACR := hmctspublic
-ACR_SUBSCRIPTION := DCD-CFT-Sandbox
-AKS_RESOURCE_GROUP := cnp-aks-sandbox-rg
-AKS_CLUSTER := cnp-aks-sandbox-cluster
+ACR_SUBSCRIPTION := DCD-CFTAPPS-DEV
+AKS_RESOURCE_GROUP := cft-preview-00-rg
+AKS_CLUSTER := cft-preview-00-aks
+CONTEXT := $(k config get-contexts | grep ${AKS_CLUSTER} | wc -l)
```

Comment:
is this used?

Reply:
No, was testing out conditions so it wouldn't always need to authenticate to azure with the device code but couldn't get it to work. Meant to remove that